rotkehlchen/globaldb/upgrades/v7_v8.py

```diff
@@ -0,0 +1,76 @@
+from typing import TYPE_CHECKING, Final
+
+from rotkehlchen.logging import enter_exit_debug_log
+from rotkehlchen.utils.misc import ts_now
+
+if TYPE_CHECKING:
+    from rotkehlchen.db.drivers.gevent import DBConnection, DBCursor
+
+
+ASSETS_TO_WHITELIST: Final = (
```

Comment:
Hmmm few questions here.

1. The ones that do not have a url why were they marked as spam?
2. What is the point of this? Just adding them to whitelist without putting them as exceptions in the algorithm, won't it still create problems?

Reply:
The other that got detected as spam is because they use `-` in the name that is one of those (not sure which one) and is in the list of punctuation symbols that the regex catches.

I need to create an exception in bugfixes to skip that symbol if present

![image](https://github.com/rotki/rotki/assets/5068010/8ee4b68b-7445-41ce-916e-c5e4570c205b)

once they are whitelisted the logic won't add them as spam because they won't reach the algorithm
the exceptions are in bugfixes because we don't have the globaldb upgrade there

I'm going to change in this line \p{P} to \p{Po} https://www.compart.com/en/unicode/category/Po

To reduce this broader catch. But in bugfixes
https://www.compart.com/en/unicode/category/Po
[Find all Unicode Characters from Hieroglyphs to Dingbats – Unicode ...](https://www.compart.com/en/unicode/category/Po)
All Unicode Symbols with Names and Descriptions on One Page
https://github.com/rotki/rotki/blob/b2799beee78006de22bb02e473b4772c4cc2208a/rotkehlchen/assets/utils.py#L48